pkg/importer/registry-datasource.go

```diff
@@ -82,7 +82,7 @@ func (rd *RegistryDataSource) Transfer(path string) (ProcessingPhase, error) {
 	rd.imageDir = filepath.Join(path, containerDiskImageDir)
 
 	klog.V(1).Infof("Copying registry image to scratch space.")
-	err = image.CopyRegistryImage(rd.endpoint, path, containerDiskImageDir, rd.accessKey, rd.secKey, rd.certDir, rd.insecureTLS)
+	err = image.CopyRegistryImage(rd.endpoint, path, "^"+containerDiskImageDir+"/.+", rd.accessKey, rd.secKey, rd.certDir, rd.insecureTLS)
```

Comment:
Why is this now a regex? I am just curious.

Reply:
Hm..
I might have overthought it and made it more universal that we need. 

Maybe I should just treat `containerDiskImageDir` as a simple path prefix... which is what the regex does.